tda/client/base.py

```diff
@@ -819,7 +819,7 @@ def __normalize_start_and_end_datetimes(self, start_datetime, end_datetime):
 
 
     def get_price_history_every_minute(
-            self, symbol, *, start_datetime=None, end_datetime=None):
+            self, symbol, *, start_datetime=None, end_datetime=None, need_extended_hours_data=True):
```

Comment:
I should have been clear, let's leave the default value as `None` to be the same as the main function. Also please insert a line break to come in under 80 chars.